rubocop/rubocop_rails.yml

```diff
@@ -3,11 +3,34 @@ inherit_from: rubocop.yml
 AllCops:
   Exclude:
     - db/schema.rb
+    - 'bin/**/*'
+    - 'config/**/*'
     - 'vendor/**/*'
 
 Rails:
   Enabled: true
 
+Metrics/BlockLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Metrics/LineLength:
+  Exclude:
+    - 'db/seeds.rb'
+
 Metrics/MethodLength:
   Exclude:
     - 'db/migrate/**/*'
+
+Metrics/ModuleLength:
+  Exclude:
+    - 'db/seeds.rb'
+    - 'spec/**/*'
+
+Style/StringLiterals:
```

Comment:
Note that there is an [open issue](https://github.com/TheGnarCo/gnarails/issues/18) in gnarails so that projects generated via that template have the provided code adhere to our coding standards defined here out of the box.